lib/new-admin/funding.js

```diff
@@ -51,7 +51,9 @@ function getSingleCoinFunding (settings, fiatCode, cryptoCode) {
       const rate = (rates.ask.add(rates.bid)).div(2)
       const fundingConfirmedBalance = fundingRec.fundingConfirmedBalance
       const fiatConfirmedBalance = computeFiat(rate, cryptoCode, fundingConfirmedBalance)
-      const pending = fundingRec.fundingPendingBalance
+      const pending = cryptoCode === 'ETH'
+        ? fundingRec.fundingPendingBalance - fundingRec.fundingConfirmedBalance
```

Comment:
I think this should be done at the wallet plugin level. This might also be different for different ETH wallets.

Reply:
I also agree. The main reason I went with this solution, was to avoid an extra request on Infura, but thinking about it, our request cache should help us with this